components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -359,14 +359,12 @@ export class GithubApp {
         //
         const installation = await this.appInstallationDB.findInstallation("github", String(installationId));
         if (!installation) {
-            log.error("Prebuilt requested from unknown GitHub app installation");
```

Comment:
Why not turn this into debug/trace? IMO log volume is no argument here (called very seldomly).